### ISSUE TITLE
🚀 Feature Request: 

### ISSUE THREAD

**Issue by jar53** [elementor/elementor#19567]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

This problem would solve a ton of issues. Adding more Dynamic tags such as pulling from a div's ID would help users with features around the entire site. Using a simple jQuery line such as jQuery('#ID'.val(); would help with a ton of features. For my instance, I wanted to change the total value for the Paypal button. The issue was that I had only three options to change the transaction total. The only way that this would work is if I would set the Paypal value to an equal minimum of $1, which I allowed for this instance. I hid the Paypal button until I used Jquery to select "<input type="hidden" name="amount" value="1">" which then would be dynamically changed when the users change their options. When they select the Paypal button, it will change the value of the button, which would be the correct amount.

### Describe the solution you'd like

If there were more options with the dynamic tags, such as selecting a Div's ID, Attributes, etc. I would not have to write my code, "Which isn't an issue." I see a simple addition being beneficial for the entirety of Elementor because pulling an ID's value could be used creatively throughout one's website! It would save time and make more options for being dynamic throughout a site.

### Describe alternatives you've considered

I've tried looking for other plugins, but they're almost $100 for one website. Why do that when I can code it myself and or Elementor can add it to their overall features!

**Comment by jamieburchell** [comment]
@jar53 Does the CSS ID field under a widget's advanced tab not give you what you need?

It's badly named really, it should be HTML ID or Element ID because it's not only for use in CSS.

**Comment by jar53** [comment]
No, it doesn't, what I'm trying to achieve is getting the value of the selector, not set the selector.

Thanks,

Jeffrey Reed Jr.

________________________________
